api/src/Page/EM/Config.php

```diff
@@ -11,7 +11,7 @@ trait Config
      */
     private function configExitIfNoMicroscopes()
     {
-        if (count($this->_get_beamlines_from_type('em')) == 0) {
+        if (count($this->_get_beamlines_from_type('em', false)) == 0) {
```

Comment:
Not sure why, but the linter didn't seem happy this was an optional input - so to get rid of the error, just set it directly.